SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -82,5 +82,21 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	/** Change 'Load' value in inspector to represent worker Frame Time instead of a fraction of target FPS.*/
 	UPROPERTY(EditAnywhere, config, Category = "Metrics", meta = (ConfigRestartRequired = false))
 	bool bUseFrameTimeAsLoad;
+
+	/** If the Development Authentication Flow is used, the client will try to connect to the cloud rather than local deployment. */
+	UPROPERTY(EditAnywhere, config, Category = "Cloud Test", meta = (ConfigRestartRequired = false))
+	bool bUseDevelopmentAuthenticationFlow;
+
+	/** The token created using 'spatial project auth dev-auth-token' */
+	UPROPERTY(EditAnywhere, config, Category = "Cloud Test", meta = (ConfigRestartRequired = false))
+	FString DevelopmentAuthenticationToken;
+
+	/** The deployment to connect when using Development Authentication Flow. If left empty, use the first available one. The deployment need to be tagged with 'dev_login'. */
```

Comment:
I didn't know that! Do you think we should add this information to the comment/tooltip?

Reply:
it might be good to have it somewhere as a comment as it's an implementation detail that we don't really mention anywhere. Just saying that we don't guarantee the order should be enough :)